transactions/reset-epoch/2024/sept-18/arguments.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "type":"UInt64",
```

Comment:
@jordanschalm I incremented the epoch by 1 but kept all other parameters same as [last week](https://github.com/onflow/service-account/blob/main/transactions/reset-epoch/2024/sept-11/arguments.json)